util/avatars.ts

```diff
@@ -0,0 +1,15 @@
+export const githubProfileAvatarUrl = (username: string): string =>
+  `https://avatars.githubusercontent.com/${username}`;
+
+export const getAcronymFromString = (str: string): string =>
+  str || str.length > 0
```

Comment:
I don't think this is correct... There's no need of explicit return type nor

```tsx
export const getAcronymFromString = (str = '') =>
  [...str.trim().match(/\b(\w)/g)].join().toLocaleUpperCase()
```

Reply:
`.match()` can return null, hence why the `[...]` spread that ensures value becomes an array. And join() ensures it becomes a string, because `[null].join()` becomes an empty string.

Reply:
Also, there's no need to call `locale` here. Plain uppercase is enough.